prow/pod-utils/gcs/metadata.go

```diff
@@ -27,11 +27,12 @@ type Started struct {
 
 // Metadata is used to mirror the metadata in the finished.json artifact.
 type Metadata struct {
-	InfraCommit string            `json:"infra-commit"`
-	Pod         string            `json:"pod"`
-	Repo        string            `json:"repo"`
-	RepoCommit  string            `json:"repo-commit"`
-	Repos       map[string]string `json:"repos"`
+	InfraCommit   string            `json:"infra-commit"`
+	Pod           string            `json:"pod"`
+	Repo          string            `json:"repo"`
+	RepoCommit    string            `json:"repo-commit"`
+	Repos         map[string]string `json:"repos"`
+	SidecarErrors map[string]error  `json:"sidecar-errors"`
```

Comment:
metadata technically can be anything, more like a `map[string]{}interface`, so probably don't need an explicit struct.

Reply:
fixed